src/components/Nav/component/CategoryList/CategoryList.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import './CategoryList.scss';
+
+const CategoryList = () => {
+  return (
+    <ul className="category-list">
+      {CATEGORY_DATA.map(info => {
+        const { id, name, engName } = info;
```

Comment:
구조분해할당 잘 하셨습니다.

```suggestion
      {CATEGORY_DATA.map(({ id, name, engName }) => {

```

이렇게 하는 방법도 있습니다. 참고로 알고만 계시면 됩니다~ 수정하지않으셔도 돼요

Reply:
감사합니다!
다음 map 함수 사용할때는 이 방법으로 사용해보겠습니다~